dasharo-compatibility/usb-type-c-semiauto.robot

```diff
@@ -0,0 +1,605 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+# TODO: maybe have a single file to include if we need to include the same
+# stuff in all test cases
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../variables.robot
+Resource            ../keywords.robot
+Resource            ../keys.robot
+
+# TODO:
+# - document which setup/teardown keywords to use and what are they doing
+# - go threough them and make sure they are doing what the name suggest (not
+# exactly the case right now)
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
```

Comment:
```suggestion
...                     Prepare Test Suite
...                     AND
...                     Skip If    not ${DOCKING_STATION_DETECT_SUPPORT}    Docking station detect tests not supported
...                     AND
```

That way we may remove all `Skip If    not ${DOCKING_STATION_DETECT_SUPPORT}` from all test cases and skip the whole suite faster

Reply:
Done, see: https://github.com/Dasharo/open-source-firmware-validation/pull/146/commits/c2e5985f0782ed73aeea0a56e7007dec3fd9f986